### PR TITLE
Update dependecies

### PR DESCRIPTION
**Pull request by beardhatcode**
Everything seems to work

Fabulous run in 8.469613s, 34.1220 runs/s, 119.2498 assertions/s.
289 runs, 1010 assertions, 0 failures, 0 errors, 0 skips

_[Original pull request](https://github.ugent.be/unipept/unipept/pull/630) by @beardhatcode on Thu Mar 22 2018 at 14:24._
_Merged by @bmesuere on Mon May 07 2018 at 16:58._

### CODE REVIEW

**Comment by beardhatcode** [comment]
To update everything (`bundle update`), I needed  to add `~> 0.3.18`  to the mysql gem
```
gem 'mysql2', '~> 0.3.18'
```

_[Original comment](https://github.ugent.be/unipept/unipept/pull/630#issuecomment-74727) by @beardhatcode on Thu Mar 22 2018 at 14:40._

**Comment by beardhatcode** [comment]
This currently runs at https://morty.ugent.be and seems to be working fine

_[Original comment](https://github.ugent.be/unipept/unipept/pull/630#issuecomment-75510) by @beardhatcode on Thu Apr 12 2018 at 18:10._

File: Gemfile [hunk 1]
```diff
@@ -30,7 +30,7 @@ gem 'therubyracer'
 
 # faster json
 # pinned on 2.17 because of an issue with 2.18: https://github.com/ohler55/oj/issues/325
-gem 'oj', '2.17.5'
+gem 'oj' #, '2.17.5'
```
Comment (bmesuere) [inline Gemfile]: @beardhatcode Is there a reason for commenting this?

_[Original comment](https://github.ugent.be/unipept/unipept/pull/630#discussion_r15239) by @bmesuere on Mon May 07 2018 at 16:28._

Comment (beardhatcode) [inline Gemfile]: I can't remember doing this on purpose. The tests give "a Fabulous run " if the comment is not in place

295 runs, 1033 assertions, 0 failures, 0 errors, 0 skips

_[Original comment](https://github.ugent.be/unipept/unipept/pull/630#discussion_r15240) by @beardhatcode on Mon May 07 2018 at 16:33._

**Pull request merged**